schemas/Field

```diff
@@ -147,6 +141,69 @@
             "schema:sameAs": {
                 "@id": "reproterms:inputType"
             }
+        },
+        {
+            "@id": "schema:isPartOf",
+            "@type": "rdf:Property",
+            "rdfs:comment": "Indicates an item that this item is part of.",
+            "rdfs:label": "isPartOf",
+            "schema:domainIncludes": {
+                "@id": "reproterms:Field"
+            },
+            "schema:inverseOf": {
+              "@id": "schema:hasPart"
+            },
+            "schema:rangeIncludes": [
+                {
+                    "@id": "reproschema:Activity"
+                },
+                {
+                    "@id": "schema:URL"
+                }
+            ],
+            "schema:sameAs": {
+                "@id": "schema:isPartOf"
+            }
+        },
+        {
+            "@id": "reproschema:responseOptions",
+            "@type": "rdf:Property",
+            "rdfs:comment": "An element (object or by URL)to describe the properties of response of the Field item.",
+            "rdfs:label": "responseOptions",
+            "schema:domainIncludes": {
+                "@id": "reproschema:Field"
+            },
+            "schema:rangeIncludes": [
+                {
+                    "@id": "reproschema:responseOptions"
+                },
+                {
+                    "@id": "schema:URL"
+                }
+            ],
+            "schema:sameAs": {
+                "@id": "reproschema:responseOptions"
+            }
+        },
+        {
+            "@id": "reproterms:inputOptions",
+            "@type": "rdf:Property",
+            "rdfs:comment": "An element (object or by URL)to describe the input options of the Field item.",
+            "rdfs:label": "inputOptions",
+            "schema:domainIncludes": {
+                "@id": "reproschema:Field"
+            },
+            "schema:rangeIncludes": [
+                {
```

Comment:
@satra can you please check this? I am not sure what this property means (or how it's different from choices). It has been used in mindlogger activities. For reference see here:
https://github.com/ReproNim/reproschema/blob/master/activities/MindLoggerDemo/items/audio-stimulus#L19

There is also a `media` key in the same example linked above. I don't know what it's usage is. I haven't included that here yet. also, I think it hasn't been represented correctly. Shouldn't it be a list of objects?

Reply:
i don't know what inputOptions would do. if we are not using it, let's remove it.

regarding the media key, i'm assuming there are two places:
1. in responseoptions to allow for choices to be an image/icon/sound
2. for a field to be a video, an audio or an image?

so 1 and 2 would require to different places to add media key. where do we add the image to the map task?

Reply:
we have used `schema:image` in `responseOptions` see here: https://github.com/ReproNim/reproschema/blob/master/activities/VoiceTask/items/guided_speech#L18

Reply:
for that specific case it doesn't seem appropriate to put it in `responseOptions`. seems more like a property of the `Field`.

perhaps we can use this property (https://schema.org/associatedMedia) at either the `Field` level or the `options` level

Reply:
ok, I'll add it at the `Field` level.